gtsfm/frontend/retriever/default_retriever.py

```diff
@@ -0,0 +1,62 @@
+"""Default image retriever for front-end.
+Authors: Travis Driver, Jon Womack
+"""
+import abc
+from typing import List, Optional, Tuple, Dict
+import itertools
+
+import numpy as np
+import cbir
+import dask
+from dask.delayed import Delayed
+from gtsam import Pose3, Cal3Bundler
+import networkx as nx
+import pickle
+
+from gtsfm.frontend.retriever.retriever_base import RetrieverBase
+import gtsfm.utils.logger as logger_utils
+from gtsfm.common.keypoints import Keypoints
+
+
+logger = logger_utils.get_logger()
+
+
+class DefaultRetriever(RetrieverBase):
```

Comment:
nice work Jonathan. Let's rename this to SequentialRetriever:
https://github.com/borglab/gtsfm/pull/393#discussion_r759431687

Reply:
Done

Reply:
Done